R/RocksDB/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RocksDB"
+version = v"8.9.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/facebook/rocksdb.git", "49ce8a1064dd1ad89117899839bf136365e49e79"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
+                  "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd rocksdb
+
+# Apply patch for arm and mingw
+if [[ "${target}" = arm* ]] || [[ "${target}" == aarch* ]]; then
```

Comment:
You are filtering out the armv7 and armv6 targets below, so you shouldn't need the first condition in this check. Also, the comment above this references mingw, so I guess that needs updating.